### PR TITLE
docs(installation) add TrueNas installation guide

### PR DESCRIPTION
**Pull request by xstar97**
There's not much to go about here; simply if using TrueNas Scale; a user only has to add the TrueCharts catalog to it and then be able to install bitmagnet.

we also manage the stack with setting up CNPG; cloudnative-postgres + offer gluetun a vpn addon that can be enabled on a per app basis.

### CODE REVIEW

**Comment by mgdigital** [comment]
Hi, I'm happy to add a link to the TrueCharts package but I can't accept the PR in this form as it reads too much like an advert and is too dominating on the installation page. I've created another PR here with a format I'd be happy with: https://github.com/bitmagnet-io/bitmagnet/pull/181

**Comment by Ornias1993** [comment]
While I agree it's too much fluff and the reference of "we" was weird.
I find it sad you deny a complete step-by-step guide for another platform.

I've no idea what the "dominating" comes down to.

**Comment by mgdigital** [comment]
> I find it sad you instantly deny a complete step-by-step guide for another platform

I'm not denying a step-by-step guide, but I've never used TrueNas or TrueCharts and it would be me who ends up having to support any instructions - just put it on the TrueCharts package page where it belongs!

File: bitmagnet.io/setup/installation.md [hunk 1]
```diff
@@ -61,6 +59,26 @@ services:
...
+
+![Bitmagnet Scale App](/assets/images/bitmagnet-scale-app.png)
+
+### Addons
+
+We offer a vpn addon by Gluetun; which is super easy to configure per app; just follow our guide [here](https://truecharts.org/manual/SCALE/guides/vpn-setup).
+
+Make sure to follow Gluetun's upstream docs for your vpn serivice; we recommend to setup gluetun by env variabes.
+
+![TrueCharts Gluetun Addon](/assets/images/truechart-gluetun-addon.png)
```
Comment (Ornias1993) [inline bitmagnet.io/setup/installation.md:80]: @xstar97 This should basically not be here.

We would also not explicitly include this in guides we make ourselves and it makes it impossible to maintain.

File: bitmagnet.io/setup/installation.md [hunk 2]
```diff
@@ -61,6 +59,26 @@ services:
...
+
+To effortlessly kickstart your journey with bitmagnet, integrate the [TrueCharts catalog](https://truecharts.org/manual/SCALE/guides/getting-started#adding-truecharts) and proceed with app installation on TrueNAS Scale.
+
+Optionally, you can configure the `TMDB API Key` by following the guidelines outlined [here](configuration.md#obtaining-a-tmdb-api-key).
+
+![Bitmagnet Config Options](/assets/images/bitmagnent-scale-app-config.png)
+
+Worth noting, the PostgreSQL database is seamlessly configured and installed alongside the app. Just ensure to meticulously adhere to the getting started guide, installing all the recommended operators before initiating the app installation process.
+
+![Bitmagnet Scale App](/assets/images/bitmagnet-scale-app.png)
```
Comment (Ornias1993) [inline bitmagnet.io/setup/installation.md:72]: This is a bit of a "useless" screenshot.
Taking a lot of space on someone elses installation page.

**Comment by Ornias1993** [comment]
> > I find it sad you instantly deny a complete step-by-step guide for another platform
> 
> I'm not denying a step-by-step guide, but I've never used TrueNas or TrueCharts and it would be me who ends up having to support any instructions - just put it on the TrueCharts package page where it belongs!

Obviously you could just point users at the fact we've support staff/volunteers ready to help out if it doesn't work.
Anyway, we'll not push you any further with trying to support your project.

**Pull request closed without merging**